evap/staff/importers.py

```diff
@@ -271,14 +273,19 @@ def read_one_enrollment(self, data):
 
     def process_course(self, course_data, sheet, row):
         course_id = course_data.name_en
+
         if course_id not in self.courses:
             if course_data.name_de in self.names_de:
                 self.errors.append(_('Sheet "{}", row {}: The German name for course "{}" already exists for another course.').format(sheet, row + 1, course_data.name_en))
             else:
                 self.courses[course_id] = course_data
                 self.names_de.add(course_data.name_de)
         else:
-            if not course_data == self.courses[course_id]:
+            if set(course_data.degree_names) != set(self.courses[course_id].degree_names) and \
+                course_data.name_de == self.courses[course_id].name_de and course_data.name_en == self.courses[course_id].name_en and course_data.type_name == self.courses[course_id].type_name and course_data.is_graded == self.courses[course_id].is_graded and course_data.responsible_email == self.courses[course_id].responsible_email:
+                self.warnings[self.W_DEGREE].append(_('Sheet "{}", row {}: The course\'s "{}" degree differs from it\'s degree in a previous row. Courses were automatically joined.').format(sheet, row+1, course_data.name_en))
+                self.courses[course_id].degree_names.extend(course_data.degree_names)
+            elif not course_data == self.courses[course_id]:
```

Comment:
I know you just copied it, but I find "not ==" strange here. @karyon @janno42 remember any reason why we did it this way? I think != would be the way to go.